positron/comms/ui-frontend-openrpc.json

```diff
@@ -152,6 +152,27 @@
 			],
 			"result": {}
 		},
+		{
+            "name": "ask_for_password",
+            "summary": "Ask the user for a password",
+            "description": "Use this for an input box where the user can input a password",
+            "params": [
+                {
+                    "name": "prompt",
+                    "description": "The prompt, such as 'Please enter your password'",
+                    "schema": {
+                        "type": "string"
+                    }
+                }
+            ],
+            "result": {
+				"required": false,
```

Comment:
The formatting in the new lines of JSON looks like a mix of tabs and spaces -- could you reformat?